packages/apollo-cli/src/commands/jbrowse/set-config.ts

```diff
@@ -0,0 +1,75 @@
+import * as fs from 'node:fs'
+import {
+  ImportJBrowseConfigChange,
+  JBrowseConfig,
+} from '@apollo-annotation/shared'
+import { Flags } from '@oclif/core'
+import { Agent, RequestInit, fetch } from 'undici'
+import { ConfigError } from '../../ApolloConf.js'
+import { BaseCommand } from '../../baseCommand.js'
+import {
+  wrapLines,
+  localhostToAddress,
+  createFetchErrorMessage,
+} from '../../utils.js'
+
+export default class SetConfig extends BaseCommand<typeof SetConfig> {
+  static summary = 'Add jbrowse configuration'
+  static description = wrapLines(
+    'Add jbrowse configuration into apollo database',
+  )
+
+  static examples = [
+    {
+      description: 'Add jbrowse configuration:',
+      command: '<%= config.bin %> <%= command.id %> -i config.json',
+    },
+  ]
+
+  static flags = {
+    'input-file': Flags.string({
+      char: 'i',
+      description: 'Input jbrowse configuration file',
+      required: true,
+    }),
+  }
```

Comment:
Can you make this a positional argument instead of a flag?

Reply:
done